### PR TITLE
Robot status

### PR DESCRIPTION
**Pull request by fmauch**
This MR aims at propagating the robot's status (robot mode and safety mode) to the user so she can act accordingly (e.g. unlock after a protective stop or power on the robot if required)

This participates to #14

### CODE REVIEW

File: ur_dashboard_msgs/action/SetMode.action [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# This action is for setting the robot into a desired mode
+# and safety mode is in a non-critical state (e.g. NORMAL or REDUCED)
```
Comment (t-schnell) [inline ur_dashboard_msgs/action/SetMode.action]: Not sure what is intended to be said here.

File: ur_robot_driver/doc/ROS_INTERFACE.md [hunk 2]
```diff
@@ -708,7 +708,7 @@ This is the actual driver node containing the ROS-Control stack. Interfaces docu
 
  * "**tf_prefix**" (default: "")
 
-    Please add description. See hardware_interface.cpp line number: 67
+    Please add description. See hardware_interface.cpp line number: 68
```
Comment (t-schnell) [inline ur_robot_driver/doc/ROS_INTERFACE.md]: I assume missing a catkin_doc comment

File: ur_robot_driver/include/ur_robot_driver/ros/robot_state_helper.h [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+ *
+ * This class logs any changes of the robot state and provides an action interface bring the robot
+ * into a required state. For modifying the robot mode it uses the dashboard server interface
+ * provided by the ur_robot_driver.
+ */
+class RobotStateHelper
+{
+public:
+  RobotStateHelper() = delete;
+  RobotStateHelper(const ros::NodeHandle& nh);
```
Comment (t-schnell) [inline ur_robot_driver/include/ur_robot_driver/ros/robot_state_helper.h]: Missing Doxygen comment

File: ur_robot_driver/doc/ROS_INTERFACE.md (hunk 2 above)
Comment (fmauch) [reply ur_robot_driver/doc/ROS_INTERFACE.md]: yes, but actually not part of this Merge

**Pull request merged**